src/bundle/Resources/views/admin/content_type/edit.html.twig

```diff
@@ -17,7 +17,7 @@
 {% block page_title_admin %}
     {% include '@ezdesign/parts/page_title.html.twig' with {
         title: 'content_type.view.edit.title'|trans({ '%name%': content_type.name })|desc('Editing Content Type: %name%'),
-        iconName: 'content-type'
+        contentType: content_type.identifier
```

Comment:
It's not `contentType`. Variables in twig are with `_`.

Reply:
There was already `iconName` in`/views/parts/page_title.html.twig`, this is why I opted for `contentType` to make it consistent within template file.

Reply:
In this case consistent is not a good way.

Reply:
OK, changed. :)